test/test-markdown-external.js

```diff
@@ -0,0 +1,19 @@
+
+
+Reveal.addEventListener( 'ready', function() {
+
+	QUnit.module( 'Markdown' );
+
+	test( 'Vertical separator', function() {
+		strictEqual( document.querySelectorAll( '.reveal .slides>section>section' ).length, 2, 'found two slides' );
```

Comment:
I just repair the wrong arguments of the highlighter's invocation. No touch
other codes.

On Fri, Jun 10, 2016 at 4:34 PM, Hakim El Hattab <notifications@github.com>
wrote:

> In test/test-markdown-external.js
> <https://github.com/hakimel/reveal.js/pull/1431#discussion_r66578926>:
>
> > @@ -0,0 +1,19 @@
> > +
> > +
> > +Reveal.addEventListener( 'ready', function() {
> > +
> > +	QUnit.module( 'Markdown' );
> > +
> > +	test( 'Vertical separator', function() {
> > +		strictEqual( document.querySelectorAll( '.reveal .slides>section>section' ).length, 2, 'found two slides' );
>
> Looks like this is giving a false positive. The md file parses into a
> presentation where "Slide 1.1", the code sample and "Slide 1.2" are all on
> the same slide. The selector still returns a length of 2 since "Slide 2",
> which is intended to be a horizontal slide, actually becomes a vertical
> slide.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/hakimel/reveal.js/pull/1431/files/16ebf2a783724527faac2036bc2f971df8dea0a5#r66578926>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAUAz3Qxxw9Eu850uo2IhXijt3KgxE_iks5qKSGjgaJpZM4GgD8b>
> .
>